src/main/java/com/mycompany/myapp/web/rest/reservation/ReservationResource.java

```diff
@@ -40,4 +40,11 @@ public ReservationResource(ReservationService reservationService){
         reservationService.deleteReservation(id);
         return ResponseEntity.ok().headers(HeaderUtil.createAlert( "reservation.deleted", id.toString())).build();
     }
+
+    @GetMapping("/request-tools")
+    @Timed
+    public ResponseEntity<Void> sendToolsRequest(@RequestParam(value = "poolId") String poolId, @RequestParam(value = "selectedTools") String[] selectedTools) throws Exception {
+        reservationService.sendToolsRequest(poolId, selectedTools);
+        return ResponseEntity.ok().headers(HeaderUtil.createAlert("request.failed", poolId)).build();
```

Comment:
why fail?

Reply:
What fails when you test it ?

Reply:
I was thinking about this `HeaderUtil.createAlert("request.failed", poolId)`. I don't get why you want to create a fail alert

Reply:
That's not a fail alert, that's a 'normal' alert with bad message - fixed in next commit